roles/parse_operator_metadata/tasks/main.yml

```
@@ -181,5 +185,13 @@
         msg: "{{ ansible_failed_result }}"
 
     - name: "Result of failure"
-      fail:
-        msg: "Ansible playbook while preparing the operator metadata failed"
+      set_fact:
+        parse_operator_metadata_results: "{{ parse_operator_metadata_results | combine({'result': 'pass', 'msg': ansible_failed_result}) }}"
```

I think `result` should be set to  `failure` here, e.g.
```suggestion
        parse_operator_metadata_results: "{{ parse_operator_metadata_results | combine({'result': 'failure', 'msg': ansible_failed_result}) }}"
```

---

Oversight, thank you